apps/researcher/src/app/[locale]/(objects)/objects/[id]/page.tsx

```diff
@@ -21,6 +21,8 @@ import {env} from 'node:process';
 import {formatDateCreated} from './format-date-created';
 import ObjectListsMenu from './object-lists-menu';
 import {SignedIn} from '@clerk/nextjs';
+import {fetcher} from '@/lib/enricher-instances';
+import {AdditionalType} from '@colonial-collections/enricher/src/definitions';
```

Comment:
I've updated the `enricher`: it now exports the definitions via `index.ts`, so you can do `import {AdditionalType} from '@colonial-collections/enricher'`